src/main/java/util/LottoNumberGenerator.java

```diff
@@ -28,7 +28,7 @@ private static void initialize() {
     public static List<Integer> generateLottoNumber() {
         Collections.shuffle(allNumbers);
 
-        List<Integer> lottoNumbers = allNumbers.subList(0, LOTTO_NUMBER_COUNT_LIMIT);
-        return lottoNumbers;
+        List<Integer> shuffleLottoNumbers = allNumbers.subList(0, LOTTO_NUMBER_COUNT_LIMIT);
```

Comment:
inline으로 줄여도 될 것 같군요!
```suggestion
        return allNumbers.subList(0, LOTTO_NUMBER_COUNT_LIMIT);
```